plugins/whoami.yaml

```diff
@@ -36,4 +36,3 @@ spec:
       https://github.com/rajatjindal/kubectl-whoami
   description: |
     This plugin show the subject that's currently authenticated as.
-
```

Comment:
That extra line should not be necessary if you fix that validation script.

Reply:
I have restored the plugins folder